Tests/LibCrypto/TestCurves.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022, stelar7 <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteBuffer.h>
+#include <LibCrypto/Curves/X25519.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(test_x25519)
+{
+    // https://datatracker.ietf.org/doc/html/rfc7748#section-6.1
+    u8 alice_private_key[] {
+        0x77, 0x07, 0x6d, 0x0a, 0x73, 0x18, 0xa5, 0x7d, 0x3c, 0x16, 0xc1, 0x72,
+        0x51, 0xb2, 0x66, 0x45, 0xdf, 0x4c, 0x2f, 0x87, 0xeb, 0xc0, 0x99, 0x2a,
+        0xb1, 0x77, 0xfb, 0xa5, 0x1d, 0xb9, 0x2c, 0x2a
+    };
+
+    u8 alice_public_key[] {
+        0x85, 0x20, 0xf0, 0x09, 0x89, 0x30, 0xa7, 0x54, 0x74, 0x8b, 0x7d, 0xdc,
+        0xb4, 0x3e, 0xf7, 0x5a, 0x0d, 0xbf, 0x3a, 0x0d, 0x26, 0x38, 0x1a, 0xf4,
+        0xeb, 0xa4, 0xa9, 0x8e, 0xaa, 0x9b, 0x4e, 0x6a
+    };
+
+    u8 bob_private_key[] {
+        0x5d, 0xab, 0x08, 0x7e, 0x62, 0x4a, 0x8a, 0x4b, 0x79, 0xe1, 0x7f, 0x8b,
+        0x83, 0x80, 0x0e, 0xe6, 0x6f, 0x3b, 0xb1, 0x29, 0x26, 0x18, 0xb6, 0xfd,
+        0x1c, 0x2f, 0x8b, 0x27, 0xff, 0x88, 0xe0, 0xeb
+    };
+
+    u8 bob_public_key[] {
+        0xde, 0x9e, 0xdb, 0x7d, 0x7b, 0x7d, 0xc1, 0xb4, 0xd3, 0x5b, 0x61, 0xc2,
+        0xec, 0xe4, 0x35, 0x37, 0x3f, 0x83, 0x43, 0xc8, 0x5b, 0x78, 0x67, 0x4d,
+        0xad, 0xfc, 0x7e, 0x14, 0x6f, 0x88, 0x2b, 0x4f
+    };
+
+    u32 coordinate[8] {
+        9
+    };
+
+    auto generate_public_key = [&coordinate](ReadonlyBytes private_key) {
+        auto maybe_input = ByteBuffer::copy(private_key);
+        if (maybe_input.is_error()) {
+            dbgln("Failed to create X25519 input buffer, OOM");
+        }
+
+        auto input = maybe_input.release_value();
+        auto curve = Crypto::Curves::X25519(input);
+        auto maybe_point = curve.compute_coordinate(coordinate);
+
+        if (maybe_point.is_error()) {
+            dbgln("Failed to create X25519 content buffer, OOM");
+        }
+
+        auto point = maybe_point.release_value();
+        return point;
+    };
+
+    auto generated_alice_public = generate_public_key({ alice_private_key, 32 });
+    dbgln("got: {:hex-dump}", generated_alice_public.span());
```

Comment:
Please don't use dbgln in test cases, they shouldn't print anything if they succeed.

Reply:
It was left-over from testing :P 
Removed now